core/src/main/java/com/linecorp/armeria/server/DefaultServiceConfigSetters.java

```diff
@@ -151,6 +157,24 @@ public ServiceConfigSetters defaultLogName(String defaultLogName) {
         return this;
     }
 
+    @Override
+    public ServiceConfigSetters blockingTaskExecutor(ScheduledExecutorService blockingTaskExecutor,
+                                                     boolean shutdownOnStop) {
+        this.blockingTaskExecutor = requireNonNull(blockingTaskExecutor,
+                                                   "ScheduledExecutorService blockingTaskExecutor");
+        this.shutdownBlockingTaskExecutorOnStop = shutdownOnStop;
+        return this;
+    }
+
+    @Override
+    public ServiceConfigSetters blockingTaskExecutor(int numThreads, boolean shutdownOnStop) {
+        checkArgument(numThreads >= 0, "numThreads: %s (expected: >= 0)", numThreads);
+        final BlockingTaskExecutor executor = BlockingTaskExecutor.builder()
+                                                                  .numThreads(numThreads)
+                                                                  .build();
+        return blockingTaskExecutor(executor, shutdownOnStop);
+    }
```

Comment:
Could we move this to `ServiceConfigSetters` and make this default method?

Reply:
> Could we move this to `ServiceConfigSetters` and make this a default method?

That doesn't make sense as its subclasses need to override it